project/Dependencies.scala

```diff
@@ -19,8 +19,8 @@ object Dependencies {
   val AkkaVersion = "2.6.20"
   val AkkaHttpVersion = "10.2.10" // Note: should at least the Akka HTTP version required by Akka gRPC
   val ScalaTestVersion = "3.2.14"
-  val JacksonVersion = "2.13.4"
-  val JacksonDatabindVersion = "2.13.4"
+  val JacksonVersion = "2.13.4.1"
+  val JacksonDatabindVersion = JacksonVersion
```

Comment:
CI failing to find other jackson dependencies. Looks like only jackson-databind has the `2.13.4.1` version?